File: service/SurfWebAutofac/Model/Dtos/Players/PlayerWRDto.cs
namespace Model.Dtos.Players;

public class PlayerWRDto
{
    /// <summary>
    ///     地图Id
    /// </summary>
    public long MapId { get; set; }

    /// <summary>
    ///     地图名称
    /// </summary>
    public string MapName { get; set; }

    /// <summary>
    ///     难度
    /// </summary>
    public string Difficulty { get; set; }

    /// <summary>
    ///     地图图片
    /// </summary>
    public string Img { get; set; }

    public List<PlayerStageDto> Stages { get; set; } = new();
}

File: service/Java/SurfWeb/SurfWeb/system/src/main/java/com/surfweb/system/dto/players/PlayerWRDto.java
package com.surfweb.system.dto.players;

import lombok.Data;

import java.util.List;

@Data
public class PlayerWRDto {

    /**
     * 地图Id
     */
    private long mapId;

    /**
     * 地图名称
     */
    private String mapName;

    /**
     * 难度
     */
    private String difficulty;

    /**
     * 地图图片
     */
    private String img;

    private List<PlayerStageDto> stages;
}
